### ISSUE TITLE
OCC commands not working

### ISSUE THREAD

**Issue by oparoz** [oparoz/galleryplus#4]
_From @Bodenseematze on January 28, 2016 15:3_

Regarding to the wiki  [https://github.com/interfasys/galleryplus/wiki/Create-Thumbnails](https://github.com/interfasys/galleryplus/wiki/Create-Thumbnails)   there should be occ commands to create / delete the thumbnails - which would be a really great feature!

But when I try the commands, I always get "there ar no commands defined in the namespace."
Neither
`occ gallery:create-thumbnails name`
nor
`occ galleryplus:create-thumbnails name`
works...

Is this a planned feature for a future version?
Or sholud this work?

I use OC 8.2.2 with galleryplus 14.3.0
##

_Copied from original issue: interfasys/galleryplus#32_

**Comment by oparoz** [comment]
That feature is not available yet. You need to use a special branch of Gallery.
I need to re-sync the wiki so that it picks up the changes upstream.

**Comment by oparoz** [comment]
_From @Bodenseematze on January 28, 2016 15:29_

What a pity - would be a real useful feature for me !

So will "galleryplus" in future versions of OC be "gallery"?

**Comment by oparoz** [comment]
Development is mostly done on Gallery, but as soon as new features are released, they're made available to older version of ownCloud.
That particular feature hasn't been released yet, but it should be in a month or so. There are not enough sponsors to pay for the development of all features in parallel, so things are done one at a time.

**Comment by oparoz** [comment]
_From @effemmeffe on June 28, 2016 8:58_

Just stumbled in this problem too and luckily before reporting I searched.
Any ETA, since the predicted month is long gone now?
No pressure, just asking.

I got this:

```
fmf@kodi:~$ sudo -u www-data php /var/www/owncloud/occ gallery:create-thumbnails --all --regenerate
[sudo] password for fmf:

  [Symfony\Component\Console\Exception\CommandNotFoundException]
  There are no commands defined in the "gallery" namespace.

fmf@kodi:~$

```

**Comment by oparoz** [comment]
This feature has been postponed and may be introduced in the core component of ownCloud and Nextcloud.

**Comment by oparoz** [comment]
_From @effemmeffe on June 28, 2016 13:19_

Thanks.
Is there some official ETA or roadmap on next versions somewhere to be read? Again, no pressure, just for information.

Plus: Nextcloud?

**Comment by oparoz** [comment]
ownCloud will try to do something for 9.2 and it could then be part of [Nextcloud 11](https://nextcloud.com/)

**Comment by oparoz** [comment]
_From @effemmeffe on June 28, 2016 13:29_

Little OT:
What are the difference between owncloud and nextcloud?
Why should I choose one instead of the other?
I just discovered ownclound and installed it, should I investigate nextcloud or it's ok to stay with owncloud?

**Comment by oparoz** [comment]
> What are the difference between owncloud and nextcloud?

Philosophy.
- ownCloud offers an open core and enterprise only features. They own the code
- Nextcloud is full open

You should read this:
https://nextcloud.com/who-is-nextcloud/

I will focus on Nextcloud.

**Comment by oparoz** [comment]
_From @effemmeffe on June 28, 2016 13:38_

Damn, I just finished to install owncloud... :(
And I don't see a nextcloud package for ubuntu.
Is there some documentation to switch between owncloud and nexcloud?

**Comment by oparoz** [comment]
> Is there some documentation to switch between owncloud and nexcloud?

I suggest you check here:
https://help.nextcloud.com/

**Comment by oparoz** [comment]
_From @effemmeffe on June 28, 2016 13:43_

Thanks.

**Comment by Huster-at-github** [comment]
Any news? - I really need the feature "./occ gallery:create-thumbnails --all"

**Comment by Huster-at-github** [comment]
Any news? - I really need the feature "./occ gallery:create-thumbnails --all"

**Comment by oparoz** [comment]
Install Nextcloud 11 and use this app:
https://apps.nextcloud.com/apps/previewgenerator

**Comment by Huster-at-github** [comment]
For Gallery thumbnails!?!?

**Comment by oparoz** [comment]
Project is not longer active. The successor is included in [Nextcloud](https://nextcloud.com/).
